app/kumactl/cmd/install/render_files.go

```diff
@@ -6,13 +6,17 @@ import (
 	"github.com/Masterminds/sprig"
 	"github.com/pkg/errors"
 	"io"
+	"strings"
 	"text/template"
 )
 
 func renderFiles(templates []data.File, args interface{}, newRenderer func(data.File) (templateRenderer, error)) ([]data.File, error) {
 	renderedFiles := make([]data.File, len(templates))
 
 	for i, template := range templates {
+		if !strings.HasSuffix(template.Name, ".yaml") { // we should only render YAML files
```

Comment:
Why do we need to hardcode this constraint in here instead of passing in a pre-filtered list of `templates` ?

The rendering logic should be reusable for any text files.